src/SourceGenerators/Uno.UI.SourceGenerators.Tests.Shared/Given_XBindRewriter.cs

```diff
@@ -15,16 +15,16 @@ public class Given_XBindRewriter
 		[DataRow("ctx", "MyProperty.A", "ctx.MyProperty.A")]
 		[DataRow("ctx", "MyProperty", "ctx.MyProperty")]
 		[DataRow("ctx", "MyStaticProperty", "MyStaticProperty")]
-		[DataRow("ctx", "MyStaticMethod()", "MyStaticMethod()")]
+		[DataRow("ctx", "MyStaticMethod()", "ctx.MyStaticMethod()")]
 		[DataRow("ctx", "MyProperty.A.ToLower()", "ctx.MyProperty.A.ToLower()")]
-		[DataRow("ctx", "System.String.Format('{0:X8}', a.Value)", "System.String.Format('{0:X8}', ctx.a.Value)")]
-		[DataRow("ctx", "Static.MyFunction(42.0)", "Static.MyFunction(42.0)")]
-		[DataRow("ctx", "Static.MyFunction(true)", "Static.MyFunction(true)")]
-		[DataRow("ctx", "Static.MyFunction(MyProperty)", "Static.MyFunction(ctx.MyProperty)")]
-		[DataRow("ctx", "MyNameSpace.Static2.MyProperty", "MyNameSpace.Static2.MyProperty")]
+		[DataRow("ctx", "global::System.String.Format('{0:X8}', a.Value)", "global::System.String.Format('{0:X8}', ctx.a.Value)")]
+		[DataRow("ctx", "Static.MyFunction(42.0)", "ctx.Static.MyFunction(42.0)")]
```

Comment:
What is `ctx` in this case? Is this `global::` ?

Reply:
@jeromelaban It's the `Page` or `UserControl`.

So from my testing, `{x:Bind SomethingStaticInPage}` doesn't work in WinUI. The generated code will be `thePageInstance.SomethingStaticInPage` and then you get `error CS0176: Member cannot be accessed with an instance reference; qualify it with a type name instead`

WinUI appears to use static access only if there is a namespace prefix, e.g, `local:ClassName.StaticMethodName()`.

Reply:
I see, that makes sense. Thanks.